sound/soc/sof/Kconfig

```diff
@@ -144,6 +144,14 @@ config SND_SOC_SOF
 	  module dependencies but since the module or built-in type is decided
 	  at the top level it doesn't matter.
 
+config SND_SOC_SOF_PROBE_WORK_QUEUE
+	bool
+	help
+          This option is not user-selectable but automagically handled by
+	  'select' statements at a higher level
+	  When selected, the probe is handled in two steps, for example to
+	  avoid lockdeps if request_module is used in the probe.
```

Comment:
I might have spent too much time in Android, I think it used to be, that to hide Kconfig options and make them non-selectable one had to remove the "help" block, and if a "help" block is there, it will show up for user to click. Is it different now?